frontend/src/lib/forms/index.ts

```diff
@@ -3,18 +3,21 @@ import SubmitButton from './SubmitButton.svelte';
 import Form from './Form.svelte';
 import FormField from './FormField.svelte';
 import FormError from './FormError.svelte';
-import Input from './Input.svelte';
+import Input from './InputFormField.svelte';
```

Comment:
I don't like this, we now have 2 custom `Input` components. One is a form field input with labels and stuff, another is just a bare input. But when writing new code it would be really easy to chose the wrong one. Maybe rename the bare input to SimpleInput or PlainInput. We could also rename the current Input to FormInput or something, but it's not my favorite. It also means if you need to use both on the same page then you need to use an alias for one.

Reply:
another thing, depending on the tooling, a component is assumed to be named after the file, so there could be issues having the component named differently from the file.

Reply:
👍 I named the main one back to `Input` and the "bare" one to `PlainInput`.